lib/inferno/config/boot/ig_files.rb

```diff
@@ -0,0 +1,29 @@
+require 'fileutils'
+
+Inferno::Application.register_provider(:ig_files) do
+  prepare do
+    # This process should only run once, so skipping it on workers will start it
+    # only once from the "web" process
+    next if Sidekiq.server?
+
+    target_container.start :logging
+
+    test_kit_gems = Bundler.definition.specs.select { |spec| spec.metadata['inferno_test_kit']&.casecmp? 'true' }
+
+    ig_files = test_kit_gems.flat_map do |test_kit|
+      Dir.glob(File.join(test_kit.full_gem_path, 'lib', '*', 'igs', '*.tgz'))
+    end
+
+    if test_kit_gems.none? { |gem| gem.full_gem_path == Dir.pwd }
+      ig_files += Dir.glob(File.join(Dir.pwd, 'lib', '*', 'igs', '*.tgz'))
+    end
+
+    if File.exist? File.join(Dir.pwd, 'data', 'igs')
+      ig_files.each do |source_file_path|
+        destination_file_path = File.join(Dir.pwd, 'data', 'igs', File.basename(source_file_path))
+        Inferno::Application['logger'].info("Copying #{File.basename(source_file_path)} to data/igs")
```

Comment:
would be nice if we captured the test kit name in `ig_files` so we could print it out here too, so you know where the files came from in case multiple test kits have the same name (e.g. multiple test kit just throws a package.tgz in there (ack, we should flat out block that or add a spec rule for that), or 2 decide to 'fix' the same version of us core in different ways.  I'm not suggesting we necessarily try to proactively solve that problem, but having a tad more logged here might help us figure out where to look if there are multiple of the same name.  Also, it would help to know which one 'won'.

Reply:
> decide to 'fix' the same version of us core in different ways

Any alterations to IG files could be problematic, depending on how the validator works. I could imagine that if a modified IG package is loaded and added to the package cache first, then any subsequent validators just relying on that package id and version would use the modified rather than published version.

Reply:
Yeah, 'fixing' published IG files is a gross case but unfortunately something we need to deal with if we want to provide a decent experience to users.  I suppose the more sane case is for 'unpublished' IGs, and hopefully its obvious where they come from.

If you won't want to add complexity to the code here to log where the IG comes from that's fine by me, leave it up to you.

Reply:
I already updated the logging.